### ISSUE TITLE
Weather daily summary really pessimistic

### ISSUE THREAD

**Issue by Murmele** [woheller69/omweather#70]
**Describe the bug**
When looking on the weekly forecast the weather looks really pessimistic. When checking then the detail it looks much better. Do you get the summary from om or are you calculating it in the app?

**How to Reproduce**
1. Open omWeather
2. Check weather

**Expected behavior**
The day summary forecast represents a good summary of the detailed one

**Screenshots**
![Screenshot_20231218_055147_omWeather](https://github.com/woheller69/omweather/assets/10099533/213b0f9c-1ef7-4bc3-bcb8-d74a427b2a81)
![Screenshot_20231218_055140_omWeather](https://github.com/woheller69/omweather/assets/10099533/739f568a-8745-4e24-98d8-c525a8fee8e8)
![Screenshot_20231218_055132_omWeather](https://github.com/woheller69/omweather/assets/10099533/f31ef0a3-5e9b-4f90-980b-9a0b37329b36)
![Screenshot_20231218_055121_omWeather](https://github.com/woheller69/omweather/assets/10099533/996eb566-e9f5-48fe-b3c9-a97657122d99)
![Screenshot_20231218_055114_omWeather](https://github.com/woheller69/omweather/assets/10099533/b38ce5a0-b266-4e37-ba08-887c98d2a41b)

**Environment (please complete the following information):**
 - App version: 2.4

**Comment by Murmele** [comment]
Ok I found the optimized option. I would recommend to set it as default :)

**Comment by woheller69** [comment]
The official data always provides the worst case...

**Comment by a-ba** [comment]
I came across the same issue. I checked the raw data returned and it looks very correct.

The point is that the daily temparature range reported by open-meteo is computed over the whole day (24-hour period) whereas the high/low numbers displayed in media outlets usually reflect the temperatures you experience during daylight (which is more relevant to most users: you do not decide what to wear according to the lowest temperature reached in late night).

One possible way to address this (i.e. to displays a temperatures relevant to the user's habits) would be to recompute the min and max temparature from the hourly forecast and over a customisable interval (eg: 8am-20pm).
